pkg/tbtc/registry.go

```diff
@@ -194,6 +236,21 @@ func (ws *walletStorage) saveSigner(signer *signer) error {
 	return nil
 }
 
+// archiveWallet archives the given wallet data in the underlying persistence
+// layer of the walletStorage.
+func (ws *walletStorage) archiveWallet(walletStoragePath string) error {
```

Comment:
Let's be consistent:
```suggestion
func (ws *walletStorage) archiveWallet(walletStorageKey string) error {
```

Reply:
Done in 740721253cad39bc9d76c872ddfdc55a98304518.